formazione/viste.py

```diff
@@ -512,6 +512,11 @@ def aspirante_corso_base_iscritti_aggiungi(request, me, pk):
     return 'aspirante_corso_base_scheda_iscritti_aggiungi.html', contesto
 
 
+@pagina_privata
+def aspirante_corso_base_firme(request, me, pk):
```

Comment:
⚠️Problema di sicurezza! Non vi e' alcun controllo dei permessi su questa pagina. Qualunque utente identificato in Gaia puo' scaricare l'elenco dei partecipanti a qualunque corso su Gaia. 😱 

Solo coloro aventi permessi di gestione del corso possono scaricare il foglio firme. Questo include direttori del corso, responsabili alla formazione e Presidente del Comitato.

Per fortuna, tutto questo e' gia' codificato in Gaia e ti basta copiare due righe da una qualunque altra vista, come da quella appena sotto:
```python
if not me.permessi_almeno(corso, MODIFICA):
    return redirect(ERRORE_PERMESSI)
```

Reply:
mi mancava l'acl adatta a questo contenuto

Reply:
Perdonami, ma questo non e' accettabile.

Se sto interpretando bene il tuo commento, eri a conoscenza di questa grave falla di sicurezza, ma piuttosto che chiederci come ovviare, hai deciso di fare PR lo stesso, per mandarla in produzione.

Per piacere, se non sai come fare qualcosa di tanto importante come questo in Gaia, chiedici in chat. Siamo qui per questo. Se del codice non e' pronto per andare in produzione e vuoi solo dei commenti, aggiungi `WIP` al titolo della PR.

Reply:
era WIP in realtà

Reply:
> era WIP in realtà

Va bene, ma sta' piu' attento in futuro. Assicurati di segnalare la PR come tale, se per te qualcosa e' WIP.

Stiamo scherzando con la sicurezza delle informazioni di decine di migliaia di iscritti come potenziali corsisti (60k+).

Reply:
in verità mi aspettavo varie richieste sul layout

Reply:
> in verità mi aspettavo varie richieste sul layout

😒